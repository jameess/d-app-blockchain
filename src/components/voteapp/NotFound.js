import React from 'react';
import Image from '../../images/favicon.ico';

const NotFound = () => (
    <h1 className="text-center">
        <br />
        <img width="170rem" height="200rem" role="presentation" src={Image} /><br /><br />
        This page is like a perfect software. <br />It doesn't exists!
    </h1>
);

export default NotFound;
