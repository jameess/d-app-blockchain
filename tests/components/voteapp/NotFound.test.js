import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import NotFound from '../../../src/components/voteapp/NotFound';

test('should render NotFound component', () => {
    const wrapper = shallow(<NotFound />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});