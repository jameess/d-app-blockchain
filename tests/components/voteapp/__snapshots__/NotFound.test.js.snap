exports[`test should render NotFound component 1`] = `
<h1
  className="text-center">
  <br />
  <img
    height="200rem"
    role="presentation"
    src="favicon.ico"
    width="170rem" />
  <br />
  <br />
  This page is like a perfect software. 
  <br />
  It doesn\'t exists!
</h1>
`;
